spec/features/index_page/index_filter_by_title_spec.rb

```diff
@@ -37,5 +36,12 @@
     it 'populates the search field with the current filter' do
       expect(page).to have_selector('input[name=search_term][value=title]')
     end
+
+    it 'launches help text in a modal', js: true do
+      expect(page).to_not have_text('This represents the number of visits in which the page was viewed at least once.')
+      click_link(href: "/help/?hkey=upviews")
+      expect(page).to have_selector('.gem-c-modal-dialogue__box')
+      expect(page).to have_text('This represents the number of visits in which the page was viewed at least once.')
+    end
```

Comment:
For future reference, it might be a bit clearer to keep the test changes (this one) separate from the test setup stuff, just so the commits are a bit smaller and more focused.

Reply:
Fair point - I'm just getting used to multi-commit PRs but there's clearly room for improvement.

Reply:
:100: